src/pages/[organizationSlug]/projects/[projectSlug]/assets/[assetSlug]/index.tsx

```diff
@@ -57,7 +60,7 @@ const SecurityControlCenter: FunctionComponent<Props> = () => {
   const { personalAccessTokens, onCreatePat } = usePersonalAccessToken();
   const pat = (
     personalAccessTokens.length > 0 ? personalAccessTokens[0] : null
-  ) as PatWithPrivKey | null;
+  ) as PatWithPrivKey | undefined;
```

Comment:
```suggestion
  ) as PatWithPrivKey | undefined | null
```

The pat variable might be null - actually I see no way how it can be undefined.